app/src/main/java/org/systers/mentorship/view/activities/AboutActivity.kt

```diff
@@ -56,49 +42,19 @@ class AboutActivity : AppCompatActivity(), View.OnClickListener {
             }
         }
 
-        showProgress()
-        webView.loadUrl(url)
+        val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
+        try {
+            startActivity(intent)
+        } catch (e: ActivityNotFoundException) {
+            Toast.makeText(this,R.string.activity_not_found_exception_error, Toast.LENGTH_LONG).show()
```

Comment:
I think we should redirect the user to the website if app is not found.

Reply:
That is what happens.